guides/defensive-domain-registration.md

```diff
@@ -0,0 +1,49 @@
+---
+expires: 2021-06-01
+---
+# Defensive domain registrations
+
+The MOJ and associated organisations (Executive agencies, non-departmental public bodies and so on) main varying levels of 'online presence' using domain registrations that are often a cornerstone of the organisation's identity on the public internet, for example, as the email domain for contacting other government organisations, partners and members of the public.
+
+Each MOJ organisation **must** identify a core set of internet domains it considers critical to it’s internet identity and defensively register a small number of overt variations (for example, `justice.gov.uk` may justify `justicegov.uk`, `justice.co.uk` and `justice.uk`) to protect itself, partners and members of the public from illegitimate parties pretending to be the organisation when they are not (this usually occurs as phishing emails).
```

Comment:
critical to it’s internet identity and defensively -> critical to its internet identity. Each MOJ organisation must then defensively
overt variations -> obvious variations
and `justice.uk`) to protect itself, partners and members of the public from -> and `justice.uk`). These registrations will help protect the organisation, as well as its partners and members of the public, from
not (this usually occurs as phishing emails). -> not. Failing to register these domains can cause problems, such as phishing emails using what seem to be plausible domains.